androidapp/HopSquadApp/app/src/main/java/com/hopsquad/hopsquadapp/api/WebServiceRepository.java

```diff
@@ -41,6 +42,9 @@
     // We have 2 different types of date formats, so we create 2 services.
     private static Webservice getAllBeersWebService = initializeWebService("MM/dd/yyyy");
 
+    // Same format, different service.
+    private static Webservice getUserWebService = initializeWebService("MM/dd/yyyy");
```

Comment:
Did you try using the same format web service? I had to create multiple of this just because of the format. So if they use the same they should be able to share the web service.

Reply:
Yeah it just felt wrong using the "Get all beers web service" to retrieve a user but i should have just renamed to webservice2 or something. Will fix. I wasn't very creative last night.